### PR TITLE
Add quickstart chapter to the docs

### PR DESCRIPTION
**Pull request by ikalnitsky**
Quickstart chapter covers such topics as:

* how to install holocron
* how to create a new site
* how to manage site content
* how to preview your site

### CODE REVIEW

File: docs/quickstart.rst [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+Reading the whole documentation is highly recommended. Still, if you're
+eager to get started you can follow the guide on this page.
+
+
+Installation
+~~~~~~~~~~~~
+
+Holocron is distributed as a Python package and is available on PyPI_. It
+requires Python 3.3+ and could be install through the ``pip`` tool:
```
Comment (PropheticBird) [inline docs/quickstart.rst]: could be install**ed**

File: docs/quickstart.rst [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+~~~~~~~~~~~~~~~~~~~~
+
+Well, the site is created and it contains example contents. In order to
+preview it in a browser, you can use the ``serve`` command.
+
+.. code:: bash
+
+    $ holocron serve
+
+The command starts a debug server and prints the address to the terminal,
```
Comment (PropheticBird) [inline docs/quickstart.rst]: prints address of it ?

File: docs/quickstart.rst [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+~~~~~~~~~~~~~~~~~
+
+Well, you have a directory with a Holocron-powered site, and it contains
+one post and one page. Sounds good, ha? Let's preview them in your browser.
+
+.. code:: bash
+
+    $ holocron serve
+
+The ``serve`` command starts a development server and prints an http
```
Comment (PropheticBird) [inline docs/quickstart.rst]: I think that sounds better: 
>... and **prints to a terminal** an http address where your blog is available.

File: docs/quickstart.rst [hunk 4]
```diff
@@ -0,0 +1,150 @@
...
+address where your blog is available to a terminal. In other words, you
+should just copy-paste this address from the terminal to a browser and
+that's it.
+
+But the ``serve`` command does even more - it watches for content changes
+and rebuilds blog if any. That means you don't need to run anything in order
```
Comment (PropheticBird) [inline docs/quickstart.rst]: > .. if **there are** any

File: docs/quickstart.rst [hunk 5]
```diff
@@ -0,0 +1,150 @@
...
+to preview latest changes, all you have to do is to refresh a web-page
+in your browser.
+
+
+Manage Your Content
+~~~~~~~~~~~~~~~~~~~
+
+In order to understand how to manage Holocron's content, let's take a look
+at blog skeleton content. Moreover, it's convenient since it was generated
+in previous sections.
```
Comment (PropheticBird) [inline docs/quickstart.rst]: What do you think about putting this into one sentence? 
> In order to understand how to manage Holocron's content, let's take a look at the blog skeleton 
and example content which was generated by ```init``` command.

File: docs/quickstart.rst [hunk 6]
```diff
@@ -0,0 +1,150 @@
...
+
+  .. note::
+     The filename starts with underscore because Holocron do not process any
+     content started with underscore (``_``) or dot (``.``), and we don't
+     want to see this file in the output directory.
+
+**about.mdown**
+  A sample page. Each file that could be parsed by one of enabled converters
+  (default: Markdown only) is considered as either Page or Post. Both Page
+  and Post documents  preserves their relative paths to the content directory
```
Comment (PropheticBird) [inline docs/quickstart.rst]: preserve

File: docs/quickstart.rst [hunk 7]
```diff
@@ -0,0 +1,150 @@
...
+  and will be available under almost the same path (a file extension will be
+  stripped)in the output one. I.e. ``$content/a/b/page.mdown`` will be
```
Comment (PropheticBird) [inline docs/quickstart.rst]: space between ```)``` and ```in```

File: docs/quickstart.rst [hunk 8]
```diff
@@ -0,0 +1,150 @@
...
+  available under the ``$host/a/b/page/`` url.
+
+**2014/04/20/holocron.mdown**
+  A sample post. Each file that could be parsed by one of enabled converters
+  and has the following path pattern - ``YEAR/MONTH/DAY/FILE`` - is
```
Comment (PropheticBird) [inline docs/quickstart.rst]: What do you think about this? 

> and has ``YEAR/MONTH/DAY/FILE`` path pattern is considered a post.

File: docs/quickstart.rst [hunk 9]
```diff
@@ -0,0 +1,150 @@
...
+  considered as a post.
+
+  It's processed using almost the same rules as a page, but unlike last one
+  it will appear in the feed or on the index page.
```
Comment (PropheticBird) [inline docs/quickstart.rst]: > in the feed **and** on the index page

File: docs/quickstart.rst [hunk 10]
```diff
@@ -0,0 +1,150 @@
...
+  converters is considered as a static file. It doesn't mean it won't be
+  processed at all, but will be copied into output directory "As Is".
+
+What all that means? That means
+
+* If you want to add new page, just drop it anywhere in the content folder
+  in the supported format (default: Markdown only).
+
+* If you want to add new post, just drop it anywhere in the content folder
+  but use irectory structure with date representation - ``YEAR/MONTH/DAY``.
```
Comment (PropheticBird) [inline docs/quickstart.rst]: typo: directory

File: docs/quickstart.rst [hunk 11]
```diff
@@ -0,0 +1,150 @@
...
+
+* If you want to add new picture, just drop it anywhere in the content folder,
+  and it will be copied to output directory "As Is", so you can link to it
+  without fear.
```
Comment (PropheticBird) [inline docs/quickstart.rst]: without hesitation

File: docs/quickstart.rst [hunk 12]
```diff
@@ -0,0 +1,150 @@
...
+
+* compiling content into HTML;
+* generating Atom feed;
+* generating sitemap.xml;
+* generating index page (with all posts)
+* generating index pages by tags (posts by tags)
+
+All results will be placed in the output directory (default: ``_output``),
+so you can share this folder with any web-server.
+
```
Comment (PropheticBird) [inline docs/quickstart.rst]: Maybe also add here explanation that normally users don't need to call build as ```serve`` already does it under the hood?

Comment (ikalnitsky) [inline docs/quickstart.rst]: I'm not sure. What if user wants to get HTMLs for deployment and that's it? I mean it's not necessary to preview site, the product of Holocron is a compiled site (HTML). In order to get this product user should run precisely ``build`` command.

Comment (PropheticBird) [inline docs/quickstart.rst]: OK, I reread everything again, it is good as it is now.

**Pull request merged**